docs/schema/modelling-beneficial-ownership-info.md

```diff
@@ -12,17 +12,17 @@ The concept of a 'statement' is at the heart of the Beneficial Ownership Data St
 
 Details of how an interested party controls or owns a company or other legal entity are wrapped in a Beneficial ownership statement.
 
-![A Beneficial ownership statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-1.svg)
+![A ownership or control statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-1.svg)
 
-Details of the subject of a beneficial ownership statement and its interested party are wrapped in their own statements. The beneficial ownership statement refers out to these statements, acting as a connector.
+Details of the subject of an ownership or control statement and its interested party are wrapped in their own statements. The ownership or control statement refers out to these statements, acting as a connector.
 
-![A Beneficial ownership statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-2.svg)
+![An ownership or control ownership statement block containing two 'interests': one a 60% share-holding interest, the other a 30% voting-rights interest](_assets/data-schema-model-2.svg)
```

Comment:
"An ownership or control ownership statement block" should be "An ownership or control statement block"